.github/workflows/docker_workflow.yml

```diff
@@ -28,10 +41,6 @@ jobs:
         uses: actions/checkout@v3
       - name: Setup cmake
         uses: jwlawson/actions-setup-cmake@v1.13
-      - name: Setup Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
       - name: Build examples
         run: |
           cmake -B build -DENABLE_SYCL=on
```

Comment:
ENABLE_SYCL has no effect in project's CMakeList.txt.

Reply:
right, thx, Done